### PR TITLE
+ added H.265 support

### PR DESCRIPTION
**Pull request by MikolajMGT**
Hi @mike1808!
Your library could be easily extended to H.265 support, so I created fork with that :)
Regards

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -1,4 +1,4 @@
-module github.com/mike1808/h264decoder
```
Comment (mike1808) [inline go.mod]: I think this change wasn't intentional. Could you please revert the module name?

**Comment by mike1808** [comment]
Thank you for your contribution! If want to continue improving this library, I can add you to outside collaborators.

File: go.mod (hunk 1 above)
Comment (MikolajMGT) [reply go.mod]: You're right, I forgot change it after testing. Fixed now.

**Comment by MikolajMGT** [comment]
Sure, I can contribute in some free time :) Happy Easter by the way!

**Pull request merged**